sphinx/util/docutils.py

```diff
@@ -392,8 +392,8 @@ class SphinxTranslator(nodes.NodeVisitor):
               This class is strongly coupled with Sphinx.
     """
 
-    def __init__(self, builder, document):
-        # type: (Builder, nodes.document) -> None
+    def __init__(self, document, builder):
```

Comment:
The base class of SphinxTranslator; NodeVisitor takes a document node on its instantiate. So it would be better to take a document node first.
For this reason, I determined to swap these arguments.